sqlgen/db.go

```diff
@@ -55,17 +53,26 @@ func NewDB(conn *sql.DB, schema *Schema) *DB {
 				return nil, err
 			}
 			clause, args = selectQuery.ToSQL()
+			span.SetTag("query", clause)
+			span.SetTag("batchItemCount", len(items))
+			if marshaledItems, err := json.Marshal(filters); err == nil {
+				span.SetTag("batchItems", string(marshaledItems))
```

Comment:
Agreed, I don't think we should include the actual SQL results, since they could be fairly large (e.g. trip proto blobs). Having the number of returned results should be sufficient for most purposes, and we can always add this in later if needed.